### DISCUSSION TITLE
io.on connection not triggering in local

### DISCUSSION THREAD

**Discussion by naveenkumarwittyparrot** [socketio/socket.io#3921] [Q&A]
Hi,
I am using windows 10.
io.on connection is not triggering in local machine. If i run same project in macOs local or server, it works.

Thanks

**Comment by darrachequesne** [comment]
We'll need a lot more details in order to be able to help you:

- which version (client & server) are you using?
- is there any error in your console?
- you can enable the logs with the `DEBUG` variable, i.e. `DEBUG=socket* node index.js`. Could you please provide the output?

Please also check https://socket.io/docs/v4/troubleshooting-connection-issues
